### ISSUE TITLE
add class "att.coordinated" to element <clip>

### ISSUE THREAD

**Issue by KristinaRichts** [music-encoding/music-encoding#952]
We are currently exploring the possibilities of addressing two-dimensional clips in AV media. Would it be possible to add the "att.coordinated" class to the `<clip>` element here? Like in the `<zone>` element?

**Comment by ahankinson** [comment]
Why not allow `<zone>` inside of `<clip>` instead? I think `<clip>` deals primarily with time, while `<zone>` deals primarily with space.

What is the use case for this? To allow for cropping video clips "on the fly"? Or to annotate a specific region of a video clip?

**Comment by bwbohl** [comment]
I agree with @ahankinson that some info on potential use cases would help evaluate this idea.

Adding `att.coordinated` to `<clip>` would require rephrasing the element's description because it is defined for temporal things.

Do you have insights into the ideas of the IIIF A/V working group (https://iiif.io/community/groups/av/av-tsg/)?

**Comment by KristinaRichts** [comment]
Apologies for the use cases coming only now. Yes, annotating specific regions of audio recordings or video clips would be one scenario. We are currently working on a Voigtländer example. It is a combination of a tape with a graphical score. In the case of a parallelization of the sources, it would be nice to describe what can be seen in the score on the one hand, and on the other hand, this would have to be linked with the corresponding position in the audio tape on a temporal level. 
Parallelization of different sources is also being worked on in the context of the DFG Viewer project for musical sources, including parallelization of audio with written sources. Therefore, we would appreciate input from you on how to best solve this technically. 
I briefly talked to @kepper about this and he pointed me to IIIF AV. Since I'm not that technically savvy about what this all entails, it might be better if he could leave his thoughts on it here himself? Thank you so much in advance.

**Comment by kepper** [comment]
I was actually comparing capablities of MEI and IIIF. For images, __x__ and __y__ coordinates are needed, and both MEI and IIIF handle those well. For audio, a different dimension (let's call it **t**) is needed, and that works equally well in both MEI and IIIF. For video, however, it is necessary to have the combination of all three dimensions – it might be desirable to refer to the viola player on the left during his solo in the second movement, as captured in a video recording of a given performance. As far as I understand IIIF AV covers exactly that, but with MEI as it stands, something similar isn't possible (yet). My first idea was to just add `att.coordinated` to `<clip>`, but actually, I think @ahankinson's idea is much better, as it better preserves the intention of the existing elements. It also contains a path to add more complex shapes than rectangles using SVG, if needed at some point. It might be helpful to think a bit about possible schematron constraints that should be added – like giving a warning when `<zone>` is inside `<clip>` when there is an audio mimetype (or file extension?). I would expect this to not be easy to control that, though.

**Comment by kepper** [comment]
discussed this at 2022 Berlin dev meeting. @fujinaga, @lpugin, @pe-ro, @musicEnfanthen and @kepper agree that allowing `<zone>` inside `<clip>` is the way forward.

**Comment by bwbohl** [comment]
As no pull-request implements this yet, I'm pushing this to Milestone 5.x

**Comment by musicEnfanthen** [comment]
Further pushing to 6.0

**Comment by bwbohl** [comment]
please feel free to submit a pull request – deadline is approaching ;-)
